### PR TITLE
add git remote lesson

### PR DESCRIPTION
**Pull request by jacobherrington**
refs #5386 

Feedback is welcome.

### CODE REVIEW

**Comment by KevinMulhern** [comment]
Thanks for opening this @jacobherrington, we will get it reviewed asap.

File: git/git_remotes.md [hunk 1]
```diff
@@ -1,22 +1,70 @@
 # Git Remotes
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+In this lesson, we will cover the `git remote` command and remote repositories. 
+
+A remote is like a bookmark that tells Git where the other versions of a project are stored. These versions are housed in "remote repositories."
```
Comment (KevinMulhern) [inline git/git_remotes.md]: I'm not sure about the bookmark analogy. I understand the intention. A bookmark is a connection or link to a specific place. Personally I think being explicit that a remote is essentially a link to a version of a project located on another machine would be better here. What do you think?

File: git/git_remotes.md [hunk 2]
```diff
@@ -1,22 +1,70 @@
...
+# Introduction to Remotes
+This section will give you a clear understanding of what a remote is and why we use remotes. 
+
+GitHub has an excellent definition for remotes: "A remote URL is Git's fancy way of saying 'the place where your code is stored.'" Remotes allow us to work in tandem with other developers by giving us central locations to store and share different versions of our projects.
+
+There is a subtle, but important difference between a "remote" and a "remote repository." A remote is the bookmark-like tool that developers use to tell Git where a remote repository is located. The remote repository is where the code actually lives, typically this is on a server or another computer.
+
+## Assignment
+1. Read an [intro to remote repositories](https://www.git-tower.com/learn/git/ebook/en/command-line/remote-repositories/introduction#start)
+2. Take a few minutes to watch [this overview](https://www.youtube.com/watch?v=kd4jMl_3LQE) of Git remotes and remote repositories
```
Comment (KevinMulhern) [inline git/git_remotes.md]: Nice resource 👍 💯

File: git/git_remotes.md [hunk 3]
```diff
@@ -1,22 +1,70 @@
...
+* Fork and clone a repository to your machine
+* Add a remote to a repository
+* View remotes on a local project
+* Remove a remote from a repository
+* Pull changes from a remote branch
+* Push changes to a remote branch
+* Submit a pull request to a remote repository that doesn't belong to you
+
+# Introduction to Remotes
+This section will give you a clear understanding of what a remote is and why we use remotes.
```
Comment (KevinMulhern) [inline git/git_remotes.md]: We can remove this sentence, the title of the section should be enough to communicate what the section is about 😄

File: git/git_remotes.md [hunk 4]
```diff
@@ -1,22 +1,70 @@
...
+GitHub has an excellent definition for remotes: "A remote URL is Git's fancy way of saying 'the place where your code is stored.'" Remotes allow us to work in tandem with other developers by giving us central locations to store and share different versions of our projects.
+
+There is a subtle, but important difference between a "remote" and a "remote repository." A remote is the bookmark-like tool that developers use to tell Git where a remote repository is located. The remote repository is where the code actually lives, typically this is on a server or another computer.
+
+## Assignment
+1. Read an [intro to remote repositories](https://www.git-tower.com/learn/git/ebook/en/command-line/remote-repositories/introduction#start)
+2. Take a few minutes to watch [this overview](https://www.youtube.com/watch?v=kd4jMl_3LQE) of Git remotes and remote repositories
+
+# Forking and Cloning a Remote Repository
+This section is going to cover forking and cloning Git repositories.
```
Comment (KevinMulhern) [inline git/git_remotes.md]: Same for this sentence, RE: The title being enough to communicate what the section is about

File: git/git_remotes.md [hunk 5]
```diff
@@ -1,22 +1,70 @@
...
+
+Remember forking is specific to GitHub, and cloning is a feature of Git. 
+
+## Assignment
+1. Watch [episode 1.3](https://www.youtube.com/watch?v=_NrSWLQsDL4&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV&index=3) of [The Coding Train](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw)'s [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) to learn about making forks on GitHub
+2. Watch [episode 1.6](https://youtu.be/yXT1ElMEkW8?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) of [The Coding Train](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw)'s [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) to understand cloning a remote repository
+
+# Using the git remote Command
+Here, we will discuss using the `git remote` command. First we will learn to add a new remote, then we will see how to view all remotes on a git repository.
```
Comment (KevinMulhern) [inline git/git_remotes.md]: Should we list some of the commands here to fill this section out a little more and as a handy reference?

File: git/git_remotes.md [hunk 6]
```diff
@@ -1,22 +1,70 @@
...
+Here, we will discuss using the `git remote` command. First we will learn to add a new remote, then we will see how to view all remotes on a git repository. 
 
 ## Assignment
-The list of resources the user will go through to learn about the topic of this lesson. Have no more than 5 resources, ideally no more than three.
+1. Read [GitHub](https://www.github.com)'s guide for [Adding a remote](https://help.github.com/articles/adding-a-remote/)
+2. Read [Listing Remote Branches](http://gitready.com/intermediate/2009/02/13/list-remote-branches.html) from [Nick Quaranto](http://gitready.com/)
+3. Read [GitHub](https://www.github.com)'s guide for [Removing a remote](https://help.github.com/articles/removing-a-remote/)
+
+# Working with a Remote Repository
+Finally, we are going to look at making changes to a repository and making a pull request to a remote repository that you don't own.
```
Comment (KevinMulhern) [inline git/git_remotes.md]: Should we fill this section out a bit more by briefly explaining what push and pull means and let the resources do the heavy lifting of going into more detail?

File: git/git_remotes.md [hunk 7]
```diff
@@ -1,22 +1,70 @@
...
+
+## Assignment
+1. Watch this [60-second video](https://www.youtube.com/watch?v=-uQHV9GOA0w) covering git pull and git push
+2. Take a look at [Pushing and Pulling](http://gitready.com/beginner/2009/01/21/pushing-and-pulling.html) from [Nick Quaranto](http://gitready.com/)
+3. Watch [Git & GitHub: Pull requests](https://www.youtube.com/watch?v=FQsBmnZvBdc) from [Codecourse](https://www.youtube.com/watch?v=FQsBmnZvBdc)
 
 ## Exercises
-A group of exercises (If Applicable) for the student to complete in relation to the topic taught in the lesson.
+* Follow [this tutorial]((https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github) from [DigitalOcean](https://www.digitalocean.com) to see forking, cloning, remotes, and pull requests in action
```
Comment (KevinMulhern) [inline git/git_remotes.md]: looks like theres an extra bracket on the link `[this tutorial]((https://w...`

**Comment by KevinMulhern** [comment]
Great work with this lesson @jacobherrington, it covers everything and has links to a lot of small but fantastic resources 😄 

Let me know if you have any questions about any of my comments.

File: git/git_remotes.md (hunk 7 above)
Comment (jacobherrington) [reply git/git_remotes.md]: Good catch!

File: git/git_remotes.md (hunk 1 above)
Comment (jacobherrington) [reply git/git_remotes.md]: I think that's fine. I struggled a bit with separating a remote from the concept of a remote repository. Being direct will probably be most effective.

**Comment by jacobherrington** [comment]
Thanks for the feedback @KevinMulhern I've made the revisions you suggested. 👍

File: git/git_remotes.md (hunk 5 above)
Comment (jacobherrington) [reply git/git_remotes.md]: Done, let me know if you had something else in mind for this

**Comment by KevinMulhern** [comment]
Thanks for making the changes @jacobherrington and thanks for all your work on this lesson. It looks great.

**Pull request merged**